src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -18,6 +18,8 @@ const ConnectButton = styled(ButtonPrimary)`
   width: 288px;
   font-weight: 600;
   font-size: 16px;
+  margin-left: auto;
```

Comment:
in general with these situations I recommend making the parent a flex and using justify-content: center. can you confirm this doesn't break where it's used currently?

Reply:
I tried and it breaks, but I can do this in future